### PR TITLE
Feature/users signup

### PR DESCRIPTION
**Pull request by YourSouLi5Mine**
Acceptance criteria:

- Signup creates user when validation success
- Signup won't create user when validation fails

### CODE REVIEW

File: app/assets/stylesheets/errors.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#error_explanation {
```
Comment (ricvillagrana) [inline app/assets/stylesheets/errors.scss]: Check out CSS naming conventions: https://medium.freecodecamp.org/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -9,7 +9,11 @@
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
-  <body>
-    <%= yield %>
+  <body style="height: 100%!important;">
```
Comment (ricvillagrana) [inline app/views/layouts/application.html.erb]: Avoid inline-style (I did it due to SweetAlert2 classes overrode mines, but there are better ways)

File: config/routes.rb [hunk 3]
```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'home#index'
+  resources :users
```
Comment (ricvillagrana) [inline config/routes.rb]: If you are not using all the paths, you can only get what you need by doing something like:
```ruby
resources :users, only: [:index, :show, :create]
```
Source: [Rails docs](https://edgeguides.rubyonrails.org/routing.html#configuring-the-rails-router)

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (YourSouLi5Mine) [reply app/views/layouts/application.html.erb]: Yeah, actually, i forgot to erase it hehe I was just trying to figure out what you did (with our talked i understood)

File: config/routes.rb (hunk 3 above)
Comment (YourSouLi5Mine) [reply config/routes.rb]: Thanks, I will refactor asap

**Pull request merged**